RubberduckTests/Inspections/AssignedByValParameterMakeLocalCopyQuickFixTests.cs

```diff
@@ -9,12 +9,20 @@
 using Rubberduck.Inspections.Resources;
 using RubberduckTests.Mocks;
 using System.Threading;
+using Rubberduck.UI.Refactorings;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor;
+using System;
+using System.Windows.Forms;
 
 namespace RubberduckTests.Inspections
 {
+
     [TestClass]
-    public class AssignedByValParameterMakeLocalCopyQuickFixTests
+    public class AssignedByValParameterMakeLocalCopyQuickFixTests : IAssignedByValParameterQuickFixDialogFactory
```

Comment:
The QuickFix calls "Create(...)" on the factory interface passed into the constructor.  During unit testing, the Testing class passes in a factory interface that points to itself so that it can construct a and return an interface to the MockDialog in order to support the tests.